cli/cmd/stat_summary.go

```diff
@@ -31,9 +31,12 @@ Valid resource types include:
 
 This command will hide resources that have completed, such as pods that are in the Succeeded or Failed phases.
 If no resource name is specified, displays stats about all resources of the specified RESOURCETYPE`,
-	Example: `   conduit statsummary deployments -a test
-conduit statsummary deployments all -a test
-conduit statsummary deployments hello1 -a test `,
+	Example: `  # Get all deployments in the test namespace.
+  conduit statsummary deployments -a test
+  conduit statsummary deployments all -a test
```

Comment:
I realize this wasn't your change, but I don't think this is a valid example of a statsummary command. Or rather, I think this command would only return stats for the "all" deployment in the test namespace, rather than all deployments in the namespace. Maybe we can just remove line 36?

Reply:
I checked and it looks like kubectl doesn't have an special treatment for a target of "all", so I think we should remove our support for that:

```
$ kubectl -n conduit get po all
Error from server (NotFound): pods "all" not found
```